dbms/src/Interpreters/Settings.h

```diff
@@ -111,8 +111,6 @@ struct Settings
                                                                                                                                                                                                                                         \
     M(SettingUInt64, min_bytes_to_use_direct_io, 0, "The minimum number of bytes for input/output operations is bypassing the page cache. 0 - disabled.")                                                                               \
                                                                                                                                                                                                                                         \
-    M(SettingUInt64, mark_cache_min_lifetime, 0, "If the maximum size of mark_cache is exceeded, delete only records older than mark_cache_min_lifetime seconds.")                                                                      \
```

Comment:
Please don't remove existing configurations, simply note it is deprecated and has no effects is enough.

Reply:
keep a deprecated settings now